Development/src/pages/devices/DevicesList.js

```diff
@@ -98,10 +101,10 @@ const DevicesList = props => {
                         </TableBody>
                     </Table>
                     <br />
-                    <PaginationButton label="FIRST" nextPage={nextPage} />
-                    <PaginationButton label="PREV" nextPage={nextPage} />
-                    <PaginationButton label="NEXT" nextPage={nextPage} />
-                    <PaginationButton label="LAST" nextPage={nextPage} />
+                    <ListPaginationButton label="first" />
```

Comment:
I'd like to see this as `<ListPaginationButton label="First" rel="first"/>` with `label` defaulted from `rel` in the same way as other React-Admin components default their label from `source` or `name` props. Is that doable quickly?

Reply:
And I think you're right that defining a `<Pagination/>` component that created all four (or two, if no first/last cursors provided) buttons would be a good step. Even if we don't fully fit back into the React-Admin model, https://github.com/marmelab/react-admin/blob/master/packages/ra-ui-materialui/src/list/Pagination.js could provide inspiration for the interface?